### DISCUSSION TITLE
Translation of new bundles

### DISCUSSION THREAD

**Discussion by blankse** [pimcore/pimcore#15881] [Q&A / Support] [answered]
How are the translation of the new bundles managed? Also via POEditor, right?
Where can we find the links to this POEditor Projects? In the docs I find only the projects for the core:
https://pimcore.com/docs/platform/Pimcore/Multi_Language_i18n/Admin_Translations

**Comment by kingjia90** [comment, marked as answer]
The new bundles are In the `extended` project, https://poeditor.com/join/project/XliCYYgILb (link can be found anytime in the docs and repo readme, but probably need some better description), despite not having the `_ext` suffix in the filename.



By looking at the [🔒 internal list](https://github.com/pimcore/poeditor-export-action/blob/main/config.csv), only Custom Reports and Admin UI Classic Bundle still have translation in both POEditor projects, due having the`_ext.yaml` files there. https://github.com/pimcore/pimcore/tree/11.x/bundles/CustomReportsBundle/translations , https://github.com/pimcore/admin-ui-classic-bundle/tree/1.x/translations

**Reply by blankse** [reply]
@kingjia90 Okay, I found it. Thank you!
When do the translation changes come in the repositories? Every bugfix release or only to the minor releases?

**Reply by kingjia90** [reply]
> @kingjia90 Okay, I found it. Thank you!
> 
> When do the translation changes come in the repositories? Every bugfix release or only to the minor releases?

AFAIK Only in minor releases, but we made an exception on the first release of these extracted bundle because the translation went missing/weren't ported properly during the extraction.
